app/auth/forms.py

```diff
@@ -70,4 +70,9 @@ class ResetPasswordRequestForm(FlaskForm):
     submit -- Submit button
     """
     email = StringField('Email', validators=[DataRequired(), Email()])
+    submit = SubmitField('Request Password Reset')
+
+class ResetPasswordForm(FlaskForm):
+    password = PasswordField('Password', validators=[DataRequired()])
+    password2 = PasswordField('Repeat Password', validators=[DataRequired(), EqualTo('password')])
     submit = SubmitField('Request Password Reset')
```

Comment:
Can we get rid of the word "Request" in the ResetPasswordForm() submit button. 

Makes more sense if it just says "Reset Password".

Reply:
done